Sebedius.js

```diff
@@ -344,7 +344,8 @@ class Sebedius extends Discord.Client {
 	 */
 	static async getGameFromSelection(message) {
 		const gameChoices = SUPPORTED_GAMES.map(g => [SOURCE_MAP[g], g]);
-		return await Sebedius.getSelection(message, gameChoices);
+		const language = await Sebedius.getLanguage(message);
+		return await Sebedius.getSelection(message, gameChoices, null, true, false, false, language);
```

Comment:
That is starting to be a lot of arguments.
Maybe I should turn the parameters into a destructured object? What do you think?

Reply:
Yeah, that bugged me a bit, too. Somehow i didn't think of turning it into an object, but that sounds like a good idea.

Reply:
Yes I'll do it for the next time.